### ISSUE TITLE
出现更换新的backbone后loss不下降的问题

### ISSUE THREAD

**Issue by mcmingchang** [xingyizhou/CenterNet#776]
我参考dlav的结构把vovnet和resnest加入进了训练，发现新的两个backbone在各种深度下Loss都不能很好地下降，训练coco数据集总的loss在8左右

**Comment by xingyizhou** [comment]
I haven't tried this yet. Here are two places need care: how do you initialize the new weights or load pretrained weights; how do you place BN layers.

**Comment by mcmingchang** [comment]
I built mobilenetv2 v3 in the same way, they all work well, I can provide the codes of resnest and vovnet to you.

**Comment by mcmingchang** [comment]
I tried to use dlav0_34 to train the birds, dogs, and cats of the coco dataset. The first training was very successful, but the second time Loss got stuck at 2.5. batch_size=12, lr=6.25e-5, input_shape=384*512.

**Comment by supergbl** [comment]
@mcmingchang if you use mobilenet v2,can you get a high map?  i trained use resnet18 without pretrained, but i can not reach the  result as high as show . until i used pretrained weights , i can get the result mAP， so i thiink the pretrained weight effect; is must used

**Comment by enhenghengheng** [comment]
> 我以相同的方式构建了mobilenetv2 v3，它们都运行良好，我可以为您提供resnest和vovnet的代码。

HI,How did you add Mobilenetv2,Can you share your backbone code?I'm a new bee.

**Comment by supergbl** [comment]
i just trained resdcn_18  without pretrained, you can try it, i got only about 62%mAP without pretrained weights

**Comment by mcmingchang** [comment]
https://github.com/CaoWGG/Mobilenetv2-CenterNet

**Comment by supergbl** [comment]
@enhenghengheng  i just trained resdcn_18 without pretrained, you can try it, i got only about 62%mAP without pretrained weights

**Comment by enhenghengheng** [comment]
> @enhenghengheng i just trained resdcn_18 without pretrained, you can try it, i got only about 62%mAP without pretrained weights

OK，i will try it, and thanks.

**Comment by supergbl** [comment]
@enhenghengheng   if you have result, please informe me , thanks

**Comment by enhenghengheng** [comment]
> @enhenghengheng if you have result, please informe me , thanks

Ok,no problem,if I have a better result I will show it for you.

**Comment by Bangbanggetnithecar** [comment]
> I built mobilenetv2 v3 in the same way, they all work well, I can provide the codes of resnest and vovnet to you.

Hi, I am trying to build mobilenetv2 in centernet, but my result is pretty bad. the loss stuck at 3. mAP is 0.09. can you tell me your parameter set? (learning rate, learning epoch, batch_size, wh_weight, hm_weight, off_weight) . You can contact me on qq(1106741252). Thank you~

**Comment by supergbl** [comment]
我测试了您发的&nbsp; mobilenetv2版本 您把上采样替换了，在pascal voc的精度更低啊， map只有0.33




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xingyizhou/CenterNet"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月27日(星期一) 下午4:06
收件人:&nbsp;"xingyizhou/CenterNet"<CenterNet@noreply.github.com&gt;;
抄送:&nbsp;"葛栢林"<541202608@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [xingyizhou/CenterNet] 出现更换新的backbone后loss不下降的问题 (#776)





  
我以相同的方式构建了mobilenetv2 v3，它们都运行良好，我可以为您提供resnest和vovnet的代码。
  
HI,How did you add Mobilenetv2,Can you share your backbone code?I'm a new bee.
